src/pages/console/tutorial/extend-cli.mdx

```diff
@@ -22,7 +22,34 @@ import all1 from "/src/fragments/pull-cli-studio.mdx";
 
 <Fragments fragments={{all: all1}} />
 
-## Adding features with the Amplify CLI
+<Callout>
+
+For Safari browser, the browser will provide a CLI login key. Copy the CLI login key from the browser and paste it into your terminal.
+
+</Callout>
+
+## Use component in code
+
+Now that your components are synced, navigate to a component and choose **Configure**. To get your React code, choose **Get component code**.
+
+Follow the instructions in the modal. If this is your first time setting up the UI components in your app, make sure to complete **Initial project setup** first to install all dependencies.
+
+![Screenshot showing how to get the component code](/images/studio/get-code.png)
+
+Next, run `amplify pull` to sync all your components into your local code base. A new `ui-components` folder with all the UI components is created.
+
+Import the UI component you want from `./ui-components` and then render it in your React app. For example:
+
+```js
+import { CardB } from './ui-components'
+...
+  return <div>
+    <CardB />
+  </div>
+...
+```
```

Comment:
```suggestion
```js
import { CardB } from './ui-components'
...
  return <div>
    <CardB />
  </div>
...
```
```

consider revising to include:
- the component function 
- proper return
- sample file name

```jsx
// src/Card.jsx
import { CardB } from './ui-components'

function MyCardComponent() {
  return (
    <div>
			<CardB />
		</div>
	)
}
```

Reply:
modified example and commited